observability/monitoring/managing-alerts/managing-alerts-as-an-administrator.adoc

```diff
@@ -26,7 +26,7 @@ include::modules/monitoring-getting-information-about-alerts-silences-and-alerti
 
 [role="_additional-resources"]
 .Additional resources
-* link:https://github.com/openshift/runbooks/tree/master/alerts/cluster-monitoring-operator[{cmo-full} runbooks] ({cmo-full} GitHub repository)
+* link:https://github.com/openshift/runbooks/tree/master/alerts/cluster-monitoring-operator[GitHub {cmo-full} runbooks repository]
```

Comment:
Might be worthwhile checking with Eric Rich if linking to a GH is OK (it looks internal but sometimes security risks exist).